src/services/feed_validator.py

```
@@ -82,29 +89,35 @@
+                )
                 if attempt < 1:
                     await asyncio.sleep(self.retry_delay)
                 continue  # Try second attempt
 
         # If we get here, both attempts failed
-        return False, f"Feed validation failed after 2 attempts"
+        return False, "Feed validation failed after 2 attempts"
 
-    async def validate_feeds(self, urls: list[str]) -> dict[str, Tuple[bool, Optional[str]]]:
+    async def validate_feeds(
+        self, urls: list[str]
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)